manifests/csi-controller.yaml

```diff
@@ -150,6 +150,16 @@ spec:
                   fieldPath: spec.nodeName
             - name: CSI_ENDPOINT
               value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
+            - name: VCD_BASIC_AUTH_USERNAME
```

Comment:
Does this need to be base64 to work? If so, could we add Base64 to this name or add a comment?

Reply:
Also, this comment applies to the password

Reply:
This does not need base64